src/test/java/edu/usf/cutr/test/ResultAnalyzerTest.java

```diff
@@ -0,0 +1,384 @@
+package edu.usf.cutr.test;
+
+import edu.usf.cutr.gtfsrtvalidator.lib.validation.ValidationRules;
+import edu.usf.cutr.transitfeedqualitycalculator.ResultsAnalyzer;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.junit.Test;
+
+import java.io.FileInputStream;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.assertEquals;
+
+public class ResultAnalyzerTest {
+
+    @Test
+    public void resultAnalyzerTest() throws java.io.IOException, java.lang.NoSuchFieldException, java.lang.IllegalAccessException {
+        String directoryName = "src/test/feeds";
```

Comment:
The typical file structure for storing these files would be `src/test/resources`.  Could you please move them there and update the `directoryName` here accordingly?